src/test/java/com/browserstack/examples/tests/RunWebDriverCucumberTests.java

```diff
@@ -16,11 +16,6 @@
 import io.cucumber.testng.PickleWrapper;
 import io.cucumber.testng.TestNGCucumberRunner;
 
-/**
- * Created with IntelliJ IDEA.
- *
- * @author Anirudha Khanna
- */
```

Comment:
same

Reply:
removed since we have not followed this throughout the repos... even for this repo ... it is not consistently followed across all files. can we descope for now?